src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Sql-SecurityAlertPolicies.json

```diff
@@ -78,7 +88,7 @@
                         ""
                       ],
                       "emailAddresses": [
-                        "admin@contoso.com"
+                        "[[parameters('emailAddresses')]"
```

Comment:
@jtracey93 @JamJarchitect I'm not sure this is correct as this would put an array within an array?

Reply:
Sugar, should of spotted that. Good catch.

Can you approve https://github.com/Azure/Enterprise-Scale/pull/1122